api.planx.uk/editor/publish.test.ts

```diff
@@ -77,13 +77,14 @@ it("does not update if there are sections in an external portal", async () => {
   });
 
   await supertest(app)
-    .post("/flows/1/publish")
+    .post("/flows/1/diff")
```

Comment:
should this be it's own new test or are we intentionally dropping a test for `/publish`?

Reply:
related nit: could `/publish` and `/diff` tests live each in their own `describe` block?

Reply:
this is simply a result of moving all validation steps (including prior validation work for sections) out of `/publish` and into `/diff` - happy to update these to be describe blocks for clarity!